src/targets/gpu/include/migraphx/gpu/gemm.hpp

```diff
@@ -1,7 +1,10 @@
 #ifndef MIGRAPHX_GUARD_RTGLIB_GEMM_HPP
 #define MIGRAPHX_GUARD_RTGLIB_GEMM_HPP
 
+#include <rocblas-types.h>
```

Comment:
These includes shouldn't go into the header file.

Reply:
moved to the cpp file